rel/overlays/bin/gen-env

```diff
@@ -22,4 +22,6 @@ WIREGUARD_IPV4_NETWORK=100.64.0.0/10
 WIREGUARD_IPV4_ADDRESS=100.64.0.1
 WIREGUARD_IPV6_NETWORK=fd00::/106
 WIREGUARD_IPV4_ADDRESS=fd00::1
+
+GATEWAY_REGISTRATION_TOKEN=$(openssl rand -base64 32)
```

Comment:
Hmm we may want to make a note somewhere that this new ENV var is needed for > 0.8 -- should we add it to docs in this PR or save for a dedicated PR when we get closer to releasing?

Reply:
Will make a PR when we are closer to release with all the docs update(including this)